packages/site-demo/content/product/components/tooltip/index.mdx

```diff
@@ -1,7 +1,21 @@
 # Tooltip
 
-<DemoBlock demo="default" />
+**Tooltips display information when users hover over or focus on an element. They are positioned from this element which act as an anchor.**
```

Comment:
~They are positioned from this element which act as an anchor.~
"They are positioned using anchor elements." (same as in the popover doc)